.github/workflows/website.yml

```diff
@@ -50,11 +53,20 @@ jobs:
       run: |
         cd site/website
         yarn run build
-
+    
+    # Upload artifact if pull_request
+    - name: Archive build
+      uses: actions/upload-artifact@v1
+      with:
+        name: _site
+        path: site/website/build/
+      if: github.event_name == 'pull_request'
+    
     # Deploy the website to GitHub pages.
     - name: Deploy
       uses: peaceiris/actions-gh-pages@v3
       with:
         github_token: ${{ secrets.GITHUB_TOKEN }}
         publish_dir: ./site/website/build/GQCP
         publish_branch: gh-pages
+      if: github.event_name == 'push'
```

Comment:
Does this mean that the website is only updated on a push to develop?

Reply:
Yes